README.md

```diff
@@ -94,3 +94,13 @@ When black must reformat your file, it will show as the black pre-commit hook
 failing. When this happens you will see that the source file has been reformatted
 and is ready to be re-added to the index. Running `git commit` again should
 result in all the hooks passing and the commit actually happening.
+
+## Add support for new checkpoint types
+
+We support new checkpoint types via plug-ins. Third-party users do this by
+writing small installable packages that define and register a new checkpoint
+type.
+
+For first part development, instead of writing an external plugin, write the new checkpoint
```

Comment:
I think you mean first-party development, though I'm not super familiar with that term. Maybe just say "Alternatively, plug-ins can be added directly to the `git-theta` package by adding the checkpoint handler to checkpoints.py and adding it to the `entry_points` dict of `setup.py`"

Reply:
Done